### PR TITLE
fix quick dive link to point to docs root

### PR DESCRIPTION
**Pull request by Luap99**
The docs were moved in #92 so the link is now dead. Update it and link to the docs root were we now have the "Getting Started with Podman" guide.

### CODE REVIEW

File: static/data/features.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ const knowPodman = {
     {
       title: 'Quick dive into Podman',
       description:
-        "Hop on over to our [Podman Quickstart Guide](https://podman.io/docs/tutorials/podman_tutorial) and we'll lead you through basic Podman commands Guide and give you pointers to more learning materials and guides.",
+        "Hop on over to our [Docs](/docs) and we'll lead you through basic Podman commands Guide and give you pointers to more learning materials and guides.",
```
Comment (TomSweeneyRedHat) [inline static/data/features.ts:17]:
```suggestion
        "Hop on over to our [Docs](/docs) and we'll lead you through the basic Podman commands Guide and give you pointers to more learning materials and guides.",
```

Comment (TomSweeneyRedHat) [reply static/data/features.ts:17]: We missed this earlier, if you could touch up.

**Comment by TomSweeneyRedHat** [comment]
And it looks like you need a rebase.

**Comment by rhatdan** [comment]
@Luap99 Rebase and we can merge.

**Pull request merged**